src/components/Todo.vue

```diff
@@ -0,0 +1,193 @@
+            >mdi-pencil</v-icon
+          >
+        </v-list-item-icon>
+
+        <v-btn
+          @click="deleteTask(item.id), (snackbar = true)"
+          :loading="loading"
+          class="ma-1"
+          color="primary"
+          plain
+        >
+          Delete
+        </v-btn>
+      </v-list-item>
+    </ul>
+
+    <div class="text-center">
+      <v-snackbar v-model="snackbar">
+        {{ DeletedText }}
+
+        <template v-slot:action="{ attrs }">
+          <v-btn color="red" text v-bind="attrs" @click="snackbar = false">
+            Close
+          </v-btn>
+        </template>
+      </v-snackbar>
+    </div>
+    <div class="text-center">
+      <v-snackbar v-model="Addsnackbar">
+        {{ AddedText }}
+
+        <template v-slot:action="{ attrs }">
+          <v-btn color="red" text v-bind="attrs" @click="Addsnackbar = false">
+            Close
+          </v-btn>
+        </template>
+      </v-snackbar>
+    </div>
+    <div class="text-center">
+      <v-snackbar v-model="Editsnackbar">
+        {{ UpdatedText }}
+
+        <template v-slot:action="{ attrs }">
+          <v-btn color="red" text v-bind="attrs" @click="Addsnackbar = false">
+            Close
+          </v-btn>
+        </template>
+      </v-snackbar>
+    </div>
+  </v-app>
+</template>
+
+<script>
+import { mapState, mapActions } from "vuex";
+export default {
+  name: "todo",
+
+  data() {
+    return {
+      eTask: "",
+      // loading3: false,
+      loading: false,
+      DeletedText: "Your data is successfully Deleted ",
+      AddedText: "Your data is  successfully Added ",
+      UpdatedText: "Your data is  successfully Edit ",
+      newItems: "",
+      snackbar: "",
+      Addsnackbar: "",
+      Editsnackbar: "",
+    };
+  },
+
+  computed: {
+    UpdItems() {
+      return this.$store.state.todo.items;
+    },
+
+    ...mapState("todo", ["items"]),
+  },
+
+  methods: {
+    ...mapActions("todo", ["addItems"]),
+    ...mapActions("todo", ["toEdit"]),
+    ...mapActions("todo", ["updatenewItems"]),
+
+    editTask(index) {
+      this.toEdit(index);
+      this.eTask = this.items[index].name;
+    },
+    saveTask(index) {
+      this.updatenewItems({ index: index, newItems: this.eTask });
+    },
+
+    handleAddItems() {
+      if (this.newItems && this.newItems.trim().length > 0) {
+        {
+          this.addItems({
+            Addsnackbar: true,
+            isEdit: false,
+            name: this.newItems,
+            id: this.items.length + 1,
+          });
+        }
+      }
+      this.newItems = "";
+    },
+
+    deleteTask(id) {
+      var i = 0;
+      var rindex = 0;
+      for (i = 0; i < this.items.length; i++) {
+        if (this.items[i].id == id) {
+          rindex = i;
+        }
+      }
+      this.items.splice(rindex, 1);
+    },
+  },
+};
+</script>
+
+
+<style scoped>
+.box {
+  max-width: 600px;
+  border: 1px solid rebeccapurple;
+  margin: auto;
+}
+.input {
```

Comment:
remove unused css and try not to use css separately as you can get classes for alignments and margins